clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -794,6 +796,13 @@ void BackendConsumer::DontCallDiagHandler(const DiagnosticInfoDontCall &D) {
                               ? diag::err_fe_backend_error_attr
                               : diag::warn_fe_backend_warning_attr)
       << llvm::demangle(D.getFunctionName()) << D.getNote();
+
+  SmallVector<std::string> InliningDecisions = D.getInliningDecisions();
```

Comment:
Could we use `StringRef` here instead?  It seems to make more sense to me.

Reply:
done in e7ed56d009a83f78ffe22eaae8256a766b0c6b4b PTAL